src/atoms/SvgIcon/SvgIconWrapper.js

```diff
@@ -1,6 +1,6 @@
 import styled from '@emotion/styled';
 
-const SvgIconWrapper = styled('div', {
+const SvgIconWrapper = styled('span', {
```

Comment:
Why this change?

Reply:
Changed SvgIconWrapper to avoid putting a div (block element) inside a button element. Best practice i think :S Should i avoid doing it?